File: seccion_3/reto.py
class curso:
    def __init__(self, nombre, creditos):
        self.nombre = nombre
        self.creditos = creditos

    def mostrar(self):
        print("Curso:", self.nombre, "Créditos:", self.creditos)

c1 = curso ("Programación", 4)
c1.mostrar()

File: seccion_3/reto.java
class Curso {
    private String nombre;
    private int creditos;

    public Curso(String nombre, int creditos) {
        this.nombre = nombre;
        this.creditos = creditos;
    }

    public void mostrar() {
        System.out.println("Curso: " + nombre + " Créditos: " + creditos);
    }

    public static void main(String[] args) {
        Curso c1 = new Curso("Programación", 4);
        c1.mostrar();
    }
}